### ISSUE TITLE
Addon import conflict with other addons

### ISSUE THREAD

**Issue by nmcclay** [DockYard/ember-composable-helpers#231]
## Version

ember-composable-helpers - v1.2.2
ember - 2.7.0
node - 5.11.0
## Test Case

```
ember addon composable-helpers-font-awesome
npm install --save ember-composable-helpers
npm install --save ember-font-awesome
```

Then add this to your `index.js` file:

```
module.exports = {
  name: 'composable-helpers-font-awesome-addon-test',

  included: function(app, parentAddon) {
    // fails ember-composable-helpers
    this._super.included(arguments);

    // fails ember-font-awesome
    // this._super.included.apply(this, arguments);
  }
};
```
## Steps to reproduce

Run `ember s`
## Expected Behavior

Addon should successfully load both nested addons.
## Actual Behavior

```
ember-composable-helpers fails with the following error:
Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
    at CoreObject.module.exports.filterHelpers (/Users/nmcclay/git/composible-test/node_modules/ember-composable-helpers/index.js:41:18)
    at CoreObject.module.exports.treeForAddon (/Users/nmcclay/git/composible-test/node_modules/ember-composable-helpers/index.js:32:17)
    at CoreObject.superWrapper (/Users/nmcclay/git/composible-test/node_modules/ember-cli/node_modules/core-object/lib/assign-properties.js:32:18)
    at CoreObject._treeFor (/Users/nmcclay/git/composible-test/node_modules/ember-cli/lib/models/addon.js:373:33)
    at CoreObject.treeFor (/Users/nmcclay/git/composible-test/node_modules/ember-cli/lib/models/addon.js:341:21)
    at /Users/nmcclay/git/composible-test/node_modules/ember-cli/lib/models/addon.js:258:34
    at Array.map (native)
    at CoreObject.eachAddonInvoke (/Users/nmcclay/git/composible-test/node_modules/ember-cli/lib/models/addon.js:256:24)
    at CoreObject.treeFor (/Users/nmcclay/git/composible-test/node_modules/ember-cli/lib/models/addon.js:340:22)
    at /Users/nmcclay/git/composible-test/node_modules/ember-cli/lib/broccoli/ember-app.js:498:20
```

**Comment by offirgolan** [comment]
@nmcclay I believe a solution to this is just change this [if](https://github.com/DockYard/ember-composable-helpers/blob/master/index.js#L17) into a `while`. Can you try that change and see if it resolves your issue?

**Comment by nmcclay** [comment]
I saw that discrepancy when comparing this with the ember-font-awesome included hook as well.  However this adjustment didn't seem to make a difference in failure behavior.  I have pushed up a [demo repo](https://github.com/nmcclay/ember-composable-helpers-font-awesome-addon-conflict) so this is easier to pull down and run locally to see if we can get a resolution.  There is [remarkably little different](https://github.com/nmcclay/ember-composable-helpers-font-awesome-addon-conflict/commit/a8546f94e5ec1dc0bfd24452f9ed8ec3dadd1195) from this repo and a vanilla addon so hopefully we can resolve this more easily by building the app and tinkering with ember-composable-helpers include hook.
